app/models/application_form.rb

```diff
@@ -21,6 +21,7 @@ class ApplicationForm < ApplicationRecord
   MINIMUM_COMPLETE_REFERENCES = 2
   MAXIMUM_REFERENCES = 10
   EQUALITY_AND_DIVERSITY_MINIMAL_ATTR = %w[sex disabilities ethnic_group].freeze
+  ENGLISH_SPEAKING_NATIONALITIES = %w[British Irish].freeze
```

Comment:
I'm interpreting the designs verbatim here. 

Should this also include American, Canadian, and Australian?